src/sentry/profiles/task.py

```diff
@@ -49,6 +54,30 @@
 UI_PROFILE_PLATFORMS = {"cocoa", "android", "javascript"}
 
 
+def _get_profiles_producer_from_topic(topic: Topic) -> KafkaProducer:
+    cluster_name = get_topic_definition(topic)["cluster"]
+    producer_config = get_kafka_producer_cluster_options(cluster_name)
+    producer_config.pop("compression.type", None)
+    producer_config.pop("message.max.bytes", None)
+    return KafkaProducer(build_kafka_configuration(default_config=producer_config))
+
+
+processed_profiles_producer = SingletonProducer(
+    partial(_get_profiles_producer_from_topic, Topic.PROCESSED_PROFILES),
```

Comment:
nit: probably a little easier to read as

```suggestion
    lambda: _get_profiles_producer_from_topic(Topic.PROCESSED_PROFILES),
```